src/layers/src/geojson-layer/geojson-layer.ts

```diff
@@ -19,7 +19,7 @@
 // THE SOFTWARE.
 
 import * as arrow from 'apache-arrow';
-import {BinaryFeatures} from '@loaders.gl/schema';
+import {BinaryFeatureCollection} from '@loaders.gl/schema';
```

Comment:
Was this module not upgraded to 4.0? If not, this looks confusing if loaders 4.0 is exporting two such similar types...

Reply:
I think it’s because the "@loaders.gl/schema": "^4.1.0-alpha.2" was missing in package.json. Should be fixed in this PR.